src/app/components/account-total-balance.tsx

```diff
@@ -1,26 +1,36 @@
 import { memo } from 'react';
 
-import { styled } from 'leather-styles/jsx';
+import { css } from 'leather-styles/css';
+import { type BoxProps, styled } from 'leather-styles/jsx';
 
 import { useTotalBalance } from '@app/common/hooks/balance/use-total-balance';
 
-import { shimmerStyles } from '../../../theme/global/shimmer-styles';
+import { shimmerStyles } from '../../shared/shimmer-styles';
+import { BalanceShimmer } from './balance/balance-shimmer';
 
-interface AccountTotalBalanceProps {
+interface AccountTotalBalanceProps extends BoxProps {
   btcAddress: string;
   stxAddress: string;
 }
+
 export const AccountTotalBalance = memo(({ btcAddress, stxAddress }: AccountTotalBalanceProps) => {
-  const { totalUsdBalance, isLoading } = useTotalBalance({ btcAddress, stxAddress });
+  const { totalUsdBalance, isLoading, isInitialLoading } = useTotalBalance({
+    btcAddress,
+    stxAddress,
+  });
 
   if (!totalUsdBalance) return null;
 
+  if (isInitialLoading) {
+    return <BalanceShimmer height="20px" />;
+  }
+
   return (
     <styled.span
+      className={css(shimmerStyles)}
```

Comment:
Can we deprecate `fontWeight={500}` below while here?

Reply:
removed here and in other places where used with `label.02`